### PR TITLE
fix various translation issues

### LINKED ISSUE #562: 6603 Wording data protection acceptance on overlay
**Essential Information**
**URL:**https://gd-dev.liqd.net/en/test-video-block/
**device & browser:**
**operating system (incl. version):**
**behaviour:**
By viewing the video, you agree that your data will be transmitted to YouTube . For more information see [Data protection](https://www.google.de/intl/de/policies/privacy/).

Mit dem Aufruf des Videos erklären Sie sich einverstanden, dass Ihre Daten an YouTube übermittelt werden. Weitere Informationen finden Sie hier [Datenschutzerklärung](https://www.google.de/intl/de/policies/privacy/).

**expected behaviour:**

Remove space before full stop and use small d for data protection: By viewing the video, you agree that your data will be transmitted to YouTube. For more information see [data protection](https://www.google.de/intl/de/policies/privacy/).

"in der" instead "hier": Mit dem Aufruf des Videos erklären Sie sich einverstanden, dass Ihre Daten an YouTube übermittelt werden. Weitere Informationen finden Sie in der [Datenschutzerklärung](https://www.google.de/intl/de/policies/privacy/).

**Additional information if relevant**
**user role:**
**important screensize:**
**Comment/Question:**

**Screenshot if design related**:
Since issues are public, make sure to not display personal or secret data.

### PR DESCRIPTION
**Pull request by goapunk**
closes #562 and some more issues from weblate

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -116,8 +116,8 @@ lint-fix:
 
 .PHONY: po
 po:
-	$(VIRTUAL_ENV)/bin/python manage.py makemessages --all --extension html,email,py --ignore venv --ignore node_modules
-	$(VIRTUAL_ENV)/bin/python manage.py makemessages --all -d djangojs --ignore venv --ignore node_modules
+	$(VIRTUAL_ENV)/bin/python manage.py makemessages --all --extension html,email,py --no-obsolete --ignore venv --ignore node_modules
```
Comment (goapunk) [inline Makefile:119]: we don't want the obsolete strings at the end of the file, right? this way we don't have to delete them manually

**Review by phillimorland — APPROVED** [review]
code looking good, cannot install locally right now so won't merge till someone can check it

File: Makefile (hunk 1 above)
Comment (fuzzylogic2000) [reply Makefile:119]: No, but we would like to notice when we loose strings... I always check there for the js strings.

Comment (goapunk) [reply Makefile:119]: hmm, but you would see that in the diff anyway? I guess it's a bit harder to spot though. Now that we found out the reaason for the disappearing js translations it shouldn't happen anymore hopefully? In any case: should I remove this?

Comment (fuzzylogic2000) [reply Makefile:119]: Yes, I would like to keep the obsolete strings to make sure we don't loose translations accidentally. But maybe we only need to be sure for the js files. So only remove it from the next line?
What do you think @Rineee and @phillimorland ?

**Review by fuzzylogic2000 — APPROVED** [review]
The translations work! Afer a decision on the other thing, we can merge.

File: Makefile (hunk 1 above)
Comment (phillimorland) [reply Makefile:119]: I hadn't thought of the obsolete strings, I mostly check in the diff but maybe leaving them for js until we have brought the codebase up to scratch would be good

Comment (goapunk) [reply Makefile]: ok, I removed the commit

**Review by phillimorland — APPROVED** [review]
coolio!

**Pull request merged**